modules/security/src/main/java/io/oasp/module/security/common/base/accesscontrol/SimpleAccessControlBasedAuthenticationProvider.java

```diff
@@ -1,39 +0,0 @@
-package io.oasp.module.security.common.base.accesscontrol;
-
-import java.security.Principal;
-import java.util.Set;
-
-import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
-import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.userdetails.User;
-
-/**
- * This is a simple implementation of {@link AbstractAccessControlBasedAuthenticationProvider}.
- *
- * @author hohwille
- */
-public class SimpleAccessControlBasedAuthenticationProvider extends
```

Comment:
Why do we have to remove this? This will break compatibility if OASP users are using this and now upgrade. Are you aware of this?